src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetPersonServiceTest.kt

```diff
@@ -25,59 +25,107 @@ internal class GetPersonServiceTest(
+  describe("by search criteria") {
+    val firstName = "Bruce"
+    val lastName = "Wayne"
 
-    verify(nomisGateway, VerificationModeFactory.times(1)).getPerson(id)
-  }
+    it("returns person(s) from Prisoner Offender Search") {
+      getPersonService.execute(firstName, lastName)
 
-  it("retrieves a person from Prisoner Offender Search") {
-    getPersonService.execute(id)
+      verify(prisonerOffenderSearchGateway, VerificationModeFactory.times(1)).getPersons(firstName, lastName)
+    }
 
-    verify(prisonerOffenderSearchGateway, VerificationModeFactory.times(1)).getPerson(id)
-  }
+    it("returns person(s) from Probation Offender Search") {
+      getPersonService.execute(firstName, lastName)
 
-  it("retrieves a person from Probation Offender Search") {
-    getPersonService.execute(id)
+      verify(probationOffenderSearchGateway, VerificationModeFactory.times(1)).getPersons(firstName, lastName)
+    }
 
-    verify(probationOffenderSearchGateway, VerificationModeFactory.times(1)).getPerson(id)
-  }
+    it("returns person(s)") {
+      val personsFromPrisonerOffenderSearch = listOf(
+        Person("Bruce", "Wayne", middleName = "Gary")
+      )
+      val personsFromProbationOffenderSearch = listOf(
+        Person("Bruce", "Wayne", middleName = "John")
+      )
 
-  it("returns a person") {
-    val personFromNomis = Person("Billy", "Bob")
-    val personFromPrisonerOffenderSearch = Person("Sally", "Sob")
-    val personFromProbationOffenderSearch = Person("Molly", "Mob")
+      whenever(prisonerOffenderSearchGateway.getPersons(firstName, lastName)).thenReturn(personsFromPrisonerOffenderSearch)
+      whenever(probationOffenderSearchGateway.getPersons(firstName, lastName)).thenReturn(personsFromProbationOffenderSearch)
 
-    whenever(nomisGateway.getPerson(id)).thenReturn(personFromNomis)
-    whenever(prisonerOffenderSearchGateway.getPerson(id)).thenReturn(personFromPrisonerOffenderSearch)
-    whenever(probationOffenderSearchGateway.getPerson(id)).thenReturn(personFromProbationOffenderSearch)
+      val expectedResult = listOf(
+        Person(firstName, lastName, middleName = "Gary"),
+        Person(firstName, lastName, middleName = "John")
+      )
 
-    val result = getPersonService.execute(id)
+      val result = getPersonService.execute(firstName, lastName)
+      result.shouldBe(expectedResult)
+    }
 
-    val expectedResult = mapOf(
-      "nomis" to Person("Billy", "Bob"),
-      "prisonerOffenderSearch" to Person("Sally", "Sob"),
-      "probationOffenderSearch" to Person("Molly", "Mob")
-    )
+    it("returns an empty list when no person(s) are found") {
+      whenever(prisonerOffenderSearchGateway.getPersons(firstName, lastName)).thenReturn(emptyList())
+      whenever(probationOffenderSearchGateway.getPersons(firstName, lastName)).thenReturn(emptyList())
 
-    result.shouldBe(expectedResult)
+      val result = getPersonService.execute(firstName, lastName)
+      result.shouldBe(emptyList())
```

Comment:
```suggestion
      val result = getPersonService.execute(firstName, lastName)
      
      result.shouldBe(emptyList())
```

Sorry, newlines in tests are the simplest thing to help readability of tests imo. It makes it easier for me to spot the arrange, act and assert parts of a test.